lib/src/firestore.dart

```diff
@@ -115,6 +115,18 @@ class Firestore {
   /// Creates a write batch, used for performing multiple writes as a single
   /// atomic operation.
   WriteBatch batch() => new WriteBatch(nativeInstance.batch());
+
+  /// Retrieves multiple documents from Firestore.
+  Future<List<DocumentSnapshot>> getAll(List<DocumentReference> refs) async {
+    return (await promiseToFuture<List>(callMethod(
+            nativeInstance,
+            'getAll',
+            refs
+                .map((DocumentReference ref) => ref.nativeInstance)
+                .toList(growable: false))))
+        .map((nativeSnapshot) => DocumentSnapshot(nativeSnapshot, this))
+        .toList(growable: false);
+  }
```

Comment:
Unfortunately there is no way currently to call a vararg js method (I have tried passing an array and it failed) and actually there is no varargs in dart.

This note confirmed that we cannot pass an array: <https://stackoverflow.com/questions/49776395/firestore-getall-method-doesnt-accept-firebasefirestore-documentreference-ar>).

We have this already for select, endAt, startAfter... and I don't see any way (yet) to solve this issue.
Maybe one day we'll get the spread operator (it seems to be in discussion for Dart 2.2, I hope it will address js interop)

And yes no urgency to get it published (I actually have this code in my code accessing directly the nativeInstance - so it is more a copy of what I did so far that should rather be here)

Reply:
Sounds good, thanks!